openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/dto/v1/VerificationMethodDto.java

```diff
@@ -25,10 +25,6 @@ public enum VerificationMethodDto {
 
   SEAMLESS("seamless"),
 
-  AUTO("auto"),
-
-  WHATSAPP("whatsapp"),
-
   UNKNOWN_DEFAULT_OPEN_API("unknown_default_open_api");
```

Comment:
Currently yes: output of generated file to avoid unexpected value exception and this class is "internal" ( undocumented and for network protocol exchange only)
The next generator tool usage will move away the undocumented DTO pattern for Java SDK in favor of direct usage by end user and  it will generated dynamic enum, so it will won't be present anymore